presto-native-execution/presto_cpp/main/QueryContextManager.cpp

```diff
@@ -27,37 +28,6 @@ using facebook::presto::protocol::TaskId;
 
 namespace facebook::presto {
 namespace {
-// Utility function to translate a config name in Presto to its equivalent in
```

Comment:
@pramodsatya : You might want to separate just this change to move these properties to their own SystemSessionProperties file in a separate PR.

Reply:
Thanks for the suggestion @aditi-pandit, the `SessionProperty` class already contains the presto session property to velox property mapping so it could be reused here. 
Could you please confirm if this should be moved to a separate PR? In that case, since `SessionProperty::veloxConfigName_` would be unused, could I remove this class variable?

Reply:
@pramodsatya : Post the refactoring there isn't much left for this PR. We can just keep in this single PR. Thanks for looking into this.